db/migrate/20200406113832_drop_variable_lookup_key.rb

```diff
@@ -1,5 +1,5 @@
 class DropVariableLookupKey < ActiveRecord::Migration[5.2]
-  def change
+  def up
```

Comment:
i think you also need an empty `down` migration if you use `up`

Reply:
no, you don't it will call the predecessor, whats https://github.com/rails/rails/blob/157920aead96865e3135f496c09ace607d5620dc/activerecord/lib/active_record/migration.rb#L794 and that will skip the migration effectivelly.

Reply:
I can add `down` for clarity if you prefere though :)